profile.cc

```diff
@@ -1,5 +1,31 @@
 #include "profile.h"
 
-const double Profile::GetTime() {
-  return 0.0;
+const double Profile::GetTime(bool ignore) {
+  double delta_x = goal_.position - current_.position;
+  double discrim = (current_.velocity * current_.velocity) + (2 * kMaxAcceleration * delta_x);
+  
+  if (ignore) {  
+    double full_send_time = (current_.velocity + sqrt(discrim)) / kMaxAcceleration;
+  //  double full_send_time = delta_x / (.5 * (goal_.velocity + current_.velocity));
+  //  double full_send_time = (current_.velocity + sqrt(discrim)) / kMaxAcceleration;
+    return full_send_time;
+ } else {
+   // first part of trapezoid
+   double accel_time = (goal_.velocity - current_.velocity) / kMaxAcceleration;
```

Comment:
This should be able to go faster than the goal velocity as long as it slows down in time